src/packages/admin-ui-components/src/utils/use-schema.ts

```diff
@@ -32,13 +32,7 @@ export enum AdminUIFilterType {
 	TEXT = 'TEXT',
 }
 
-export enum EntityFieldType {
-	BOOLEAN = 'Boolean',
-	ID = 'ID!',
-	OPTIONAL_ID = 'ID',
-	JSON = 'JSON',
-	// TODO: This needs to be extended for the other types
-}
+export type EntityFieldType = 'Boolean' | 'custom' | 'ID!' | 'ID' | 'JSON';
```

Comment:
Curious why you removed the enum?

Reply:
It wouldn't work from the user's custom fields definition. The import was undefined, so I just went to strings.